src/db.rs

```diff
@@ -1027,6 +1027,18 @@ impl Db {
 	pub fn stats(&self) -> StatSummary {
 		self.inner.stats()
 	}
+
+	/// Add a new column with options specified by `new_column_options`.
+	pub fn add_column(options: &mut Options, new_column_options: ColumnOptions) -> Result<()> {
```

Comment:
```suggestion
	pub fn add_column(options: &mut Options, new_column_options: ColumnOptions) -> Result<()> {
		options.load_and_validate_metadata(false)?;
```
Just to ensure we don't overwrite other column contents.

Reply:
It would be even better to make sure there are no pending WAL logs first. This can be done by opening and then closing the database.

Reply:
done!

Reply:
I’ll note that in a comment as well.